controllers/propagator/propagation.go

```diff
@@ -42,6 +43,19 @@ type Propagator struct {
 	ReplicatedPolicyUpdates chan event.GenericEvent
 }
 
+type RetryableError struct {
```

Comment:
Can we just use this as a special type of error, and we check `errors.Is(RetryableError)`? I'm hesitant to change the return value away from just `error` for things

Reply:
Good idea!